### ISSUE TITLE
Add a Graphics pane in the text window for 1020 emulation

### ISSUE THREAD

**Issue by RespeQt** [RespeQt/RespeQt#3]
A third pane could be added to display the Graphics send by the Atari to the 1020 printer.

**Comment by josch1710** [comment]
I would suggest to put this into its own window, before going into the release.
The text window should only be used to capture text output and convert it to ASCII/Unicode.
The graphics should capture the whole output of the printer, whether it is text or graphics.

**Comment by RespeQt** [comment]
I agree. May I let you do this once I push my work as it is not my area of expertise?

**Comment by josch1710** [comment]
Ok, no problem.
